### PR TITLE
Added Comments in policies_glossary in reference to issue #94

### PR DESCRIPTION
**Pull request by top-gun007**
Made all comments in small caps for uniformity

### CODE REVIEW

**Comment by top-gun007** [comment]
@sandarumk please have a look and suggest any changes necessary.

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/FurtherResourcesFragment.java [hunk 1]
```diff
@@ -12,19 +12,31 @@
 import com.peacecorps.pcsa.R;
 
 /*
- * Resources
+ * resources
```
Comment (calistus) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/FurtherResourcesFragment.java]: Do you have a reason of changing this to small letter?

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/PoliciesFragment.java [hunk 2]
```diff
@@ -14,14 +14,24 @@
 import com.peacecorps.pcsa.SingleTextViewFragment;
 
 /*
- * Policies and Glossary, Main Screen
+ * policies and glossary, main screen
```
Comment (calistus) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/PoliciesFragment.java]: Any reason for changing the 'P' to small letter?

**Comment by sandarumk** [comment]
Seems that you have misunderstood my comment about consistency. @calistus I asked him in his previous PR to be consistent with the capitals. it seems that the student has replaced all the capitals with the simple letters. A better approach would be start comments with capitals rather than the simple letters changing the comments which were already there. More over, in the previous PR I suggested that you shouldn't add commits per a class. All these changes should have been done using a single commit mentioning the module. You can find lot of resources online about adding reasonable commit messages and determining when is the best time to commit.

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryFragment.java [hunk 3]
```diff
@@ -20,7 +20,7 @@
 import java.util.List;
 
 /*
- * Glossary
+ * glossary of important words related to women safety.
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryFragment.java]: Unnecessary comment. One more important thing is this app is not about the safety of only women.

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/PoliciesFragment.java [hunk 4]
```diff
@@ -14,14 +14,24 @@
...
+ * shows some extra relevant information related to peace corps and women safety
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/PoliciesFragment.java]: This is similar as the previous one. The app is not only about women safety and this comment is unnecessary.

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/FurtherResourcesFragment.java [hunk 5]
```diff
@@ -12,19 +12,31 @@
...
+ * provides various useful links on various topics like sexual violence, harassments etc.
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/FurtherResourcesFragment.java]: You don't have to go this far for the class comments. Resources is just enough.

**Comment by sandarumk** [comment]
Yeah. go ahead.

**Comment by sandarumk** [comment]
@top-gun007 yes. It looks good. There are some issues with the repo and currently I am attending to them. As soon as they are fixed and the repo is in stable state I will merge the PR.

**Comment by top-gun007** [comment]
thanks @sandarumk

**Pull request merged**